.env

```
@@ -17,4 +17,6 @@ NEXT_PUBLIC_API_BASE="https://flower.elevateegy.com/api/v1"
 
 
 NEXTAUTHURL = "http://localhost:3000"
-NEXTAUTH_SECRET="FDZaz8bV6y8V9KTiJqjTxjs5Npvvm39hi+qmQZqK8fw=" 
\ No newline at end of file
+NEXTAUTH_SECRET="FDZaz8bV6y8V9KTiJqjTxjs5Npvvm39hi+qmQZqK8fw=" 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)